docs/third-party-packages.md

```diff
@@ -11,6 +11,16 @@ Here are some of those third party packages:
 
 ## Plugins
 
+## Starlette Bridge
+
+<a href="https://github.com/tarsil/starlette-bridge" target="_blank">GitHub</a> |
+<a href="https://starlette-bridge.tarsild.io/" target="_blank">Documentation</a>
+
+With the deprecation of `on_startup` and `on_shutdown`, Starlette Bridge makes sure you can still
+use the old ways of declaring events with a particularity that internally, in fact, creates the
+`lifespan` for you. This way backwards compatibility is assured for the existing packages out there
+while maintaining the integrity of the newly `lifespan` events of `Starlette`.
```

Comment:
How do we sort the packages on this list? :rofl:

Reply:
Oh, sorry my bad 🤣. Let me update

Reply:
It was kind of a question actually, but thanks hahaha

Reply:
Well, I would say that maybe we could sort by alphabetical order. This solves the problem I was trying to solve in the closed PR. This way Starlette goes 100% lifespan while the [Starlette Bridge](https://starlette-bridge.tarsild.io/) keeps the old and new ways in sync for those who still want to use the `on_startup` and `on_shutdown` events.
This will make sure that packages can install it, keep the old ways (internally generates the lifespan) and upgrade to Starlette 1.0+ without issues.

Not sure if you want to use it as sugegstion in the future